specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/databases.json

```diff
@@ -0,0 +1,1047 @@
+          "enum": [
+            "Default",
+            "Copy",
+            "Secondary",
+            "PointInTimeRestore",
+            "Restore",
+            "Recovery",
+            "RestoreExternalBackup",
+            "RestoreExternalBackupSecondary",
+            "RestoreLongTermRetentionBackup",
+            "OnlineSecondary"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CreateMode",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "collation": {
+          "description": "The collation of the database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "maxSizeBytes": {
+          "format": "int64",
+          "description": "The max size of the database expressed in bytes.",
+          "type": "integer"
+        },
+        "sampleName": {
+          "description": "The name of the sample schema to apply when creating this database.",
+          "enum": [
+            "AdventureWorksLT",
+            "WideWorldImportersStd",
+            "WideWorldImportersFull"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SampleName",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "elasticPoolId": {
+          "description": "The resource identifier of the elastic pool containing this database.",
+          "type": "string"
+        },
+        "sourceDatabaseId": {
+          "description": "The resource identifier of the source database associated with create operation of this database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "status": {
+          "description": "The status of the database.",
+          "enum": [
+            "Online",
+            "Restoring",
+            "RecoveryPending",
+            "Recovering",
+            "Suspect",
+            "Offline",
+            "Standby",
+            "Shutdown",
+            "EmergencyMode",
+            "AutoClosed",
+            "Copying",
+            "Creating",
+            "Inaccessible",
+            "OfflineSecondary",
+            "Pausing",
+            "Paused",
+            "Resuming",
+            "Scaling"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "DatabaseStatus",
+            "modelAsString": true
+          }
+        },
+        "databaseId": {
+          "format": "uuid",
+          "description": "The ID of the database.",
+          "type": "string",
+   
```

Comment:
>currentSku [](start = 9, length = 10)

This question is out of curiosity. Is there really a `sku` property under `properties` and another under `database`?